src/test/java/eu/dissco/core/digitalspecimenprocessor/service/ProcessingServiceTest.java

```diff
@@ -255,13 +264,14 @@ void testUpdateSpecimenPartialElasticFailed() throws IOException, DisscoReposito
     var secondEvent = givenDigitalSpecimenEvent("Another Specimen");
     var thirdEvent = givenDigitalSpecimenEvent("A third Specimen");
     given(repository.getDigitalSpecimens(
-        List.of("A third Specimen", "Another Specimen", PHYSICAL_SPECIMEN_ID)))
+        List.of(PHYSICAL_SPECIMEN_ID, "Another Specimen", "A third Specimen")))
         .willReturn(List.of(givenUnequalDigitalSpecimenRecord(),
             givenDifferentUnequalSpecimen(SECOND_HANDLE, "Another Specimen"),
             givenDifferentUnequalSpecimen(THIRD_HANDLE, "A third Specimen")));
```

Comment:
What are the second and third specimen about?

Reply:
To check the batching funtionality. Need a couple of specimen. In this case the second specimen fails and needs to be pushed to the DeadLetterQueue (DLQ) but the rest is successful. So we need to roll back the version and handle version for the second specimen but not for the other two